semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/LiteralMethods.scala

```diff
@@ -1,20 +1,60 @@
 package io.shiftleft.semanticcpg.language.nodemethods
 
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.Languages
 import io.shiftleft.codepropertygraph.generated.nodes.{Literal, NewLocation}
 import io.shiftleft.semanticcpg.NodeExtension
 import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.semanticcpg.language.nodemethods.LiteralMethods.delimiters
+import org.apache.commons.lang3.StringUtils
+
+import scala.collection.immutable.HashMap
 
 class LiteralMethods(val literal: Literal) extends AnyVal with NodeExtension with HasLocation {
-  def value(typeFullName: String): String = {
-    if (literal.typeFullName == typeFullName) {
-      return "asd"
+  def innerText: Option[String] = {
+    val stringDelimiter = Cpg(literal.graph).metaData.language.headOption match {
+      case Some(language) => delimiters.getOrElse(language, List("\""))
+      case _              => "\"" :: Nil
```

Comment:
These two fallbacks just hide errors.
`metaData.language` always needs to exist.
`delimiters` needs to be updated for each frontend otherwise we use a maybe non sense default.